networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-hosted-zone-infoblox_{context}"]
+= Creating DNS records on an public hosted zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public hosted zone for Infoblox by using the Red Hat External DNS Operator.
+
+.Procedure
+
+. Optional: Add CA as trusted to ExternalDNS Operator for Infoblox if Infoblox uses a self signed certificate:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator create configmap trusted-ca-infoblox --from-file=ca-bundle.crt=/path/to/pem/encoded/infoblox/ca
+----
```

Comment:
This step is not needed, it was in the markdown doc just for the manual tests, the customers are supposed to have Infoblox with the right certificate.

Reply:
Ack'ed